otterdog/eclipse-thingweb.jsonnet

```diff
@@ -17,6 +17,20 @@ orgs.newOrg('eclipse-thingweb') {
     web_commit_signoff_required: false,
   },
   _repositories+:: [
+    orgs.newRepo('.github') {
+      allow_update_branch: false,
+      default_branch: "main",
```

Comment:
you can remove the two settings here:

- default_branch
- dependabot_security_updates_enabled

as the same values are already inherited from the default config, see its values here

https://eclipse-thingweb.github.io/.eclipsefdn/

You should see that also in the comment added to the PR from the validation workflow.

Reply:
I have removed them and thanks for the pointer to the website, it helps actually quite a lot :)